Makefile

```diff
@@ -67,50 +67,15 @@ pythonbase:
 	@cp -ap LICENSE /tmp/$(BASENAME)
 	@cp -ap NOTICE /tmp/$(BASENAME)
 
-binary: prep
-	@cd "$$(dirname "$(readlink -f "$0")")" || exit 1
-	$(PYTHON) make_bin_modules.py
-	@pyinstaller -y \
-	-p lib \
-	--add-data "functions.bash:." \
-	--add-data "LICENSE:." \
-	--add-data "NOTICE:." \
-	--add-data "README.md:." \
-	--add-data "ec2rlcore/help.yaml:ec2rlcore/" \
-	--add-data "bin:bin" \
-	--add-data "docs:docs" \
-	--add-data "example_configs:example_configs" \
-	--add-data "example_modules:example_modules" \
-	--add-data "ssmdocs:ssmdocs" \
-	--add-data "pre.d:pre.d" \
-	--add-data "mod.d:mod.d" \
-	--add-data "post.d:post.d" \
-	--add-data "lib/requests/cacert.pem:requests" \
-	--hidden-import botocore \
-	ec2rl.py
-
-	$(PYTHON) make_symlinks.py
-
-	@# Build the one-directory binary tarball
-	mv dist/ec2rl dist/$(BASENAME)
-	@echo "Creating ec2rl-binary.tgz ..."
-	tar -czf ec2rl-binary.tgz -C dist $(BASENAME)
-	sha256sum ec2rl-binary.tgz > ec2rl-binary.tgz.sha256
-	@echo "Done!"
-
 nightly: python
-	mv ec2rl.tgz ec2rl-nightly.tgz
-	mv ec2rl.tgz.sha256 ec2rl-nightly.tgz.sha256
+	cp ec2rl.tgz ec2rl-nightly.tgz
```

Comment:
Wouldn't running the nightly target leave behind an extra file here?

Reply:
So these sort of fixed a bug with making the RPM - pythonandbundlednightlyandrpm had previously been set to build rpm instead of nightlyrpm, and the cp resolved ec2rl.tgz being removed